### DISCUSSION TITLE
Support for PacBio reads

### DISCUSSION THREAD

**Discussion by harrismia** [jodyphelan/TBProfiler#238] [Ideas]
Hi Jody, 

       It would be great if TB Profiler had support for PacBio in addition to Illumina and MinION.  We are starting to do more PacBio sequencing and have a number of isolates with both PacBio and Illumina reads.  It would be great to be able to use TB Profiler on this data. 

     TB Profiler is a fantastic tool.  Thanks for all of your efforts!

Michael

**Comment by jodyphelan** [comment]
Hi @harrismia ,

Sure I can look into this. Is there any publicly available data that I could use to optimise the variant calling pipeline?

**Comment by harrismia** [comment]
Hi @jodyphelan , 

   Thanks a lot!    The following BioProjects have some publicly available data:  PRJNA719670, PRJNA480888, PRJNA436997 and PRJNA421446.  PRJEB8783, PRJEB31443, PRJEB27802 and PRJNA598991 .

**Comment by jodyphelan** [comment]
Thanks, I've had a look at these data and performed a small analysis to optimise the variant calling parameters - namely the frequency at which variants are called. As pacbio is fairly noisy compared to illumina a minimum altrnate fraction is required to minimise the false positive variants. Using Illumina as the gold standard I looked at pairs of Illumina and pacbio to characterise the performance at several cutoffs.

### ROC curve
<img width="981" alt="image" src="https://user-images.githubusercontent.com/8697224/193162276-287b2234-743f-4897-9e14-1f46611e61f9.png">

By looking at the ROC curve it looks like 0.6 is a decent tradeoff between False positive and True positive rate.

I'll add in the code to enable `--platform pacbio` over the next few days and make 0.6 the default cutoff
